client/app/bundles/comments/containers/NavigationBarContainer.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import NavigationBar from '../components/NavigationBar/NavigationBar';
+import * as commentsActionCreators from '../actions/commentsActionCreators';
+import BaseComponent from 'libs/components/BaseComponent';
+
+function stateToProps(state) {
+  // Which part of the Redux global state does our component want to receive as props?
+  if (state.$$commentsStore) {
+    return {
+      commentsCount: state.$$commentsStore.get('$$comments').size,
+      pathname: state.railsContext.pathname,
+    };
+  } else {
+    return { };
+  }
+}
+
```

Comment:
@robwise What the equivalent using https://github.com/JedWatson/react-select. Should we use it here?